maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/CCOutcomeController.java

```diff
@@ -62,15 +57,17 @@ public class CCOutcomeController {
                     schema = @Schema(implementation = ErrorDTO.class)
             )
     )
-    public ResponseEntity<RepOrderCCOutcomeDTO> create(@Parameter(description = "RepOrder CC outcome data",
-            content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
-                    schema = @Schema(implementation = RepOrderCCOutcome.class))) @RequestBody RepOrderCCOutcome repOrderCCOutCome) {
+    public ResponseEntity<RepOrderCCOutcomeDTO> create(
+            @Parameter(description = "RepOrder CC outcome data",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
+                            schema = @Schema(implementation = RepOrderCCOutcome.class))) @RequestBody RepOrderCCOutcome repOrderCCOutCome) {
```

Comment:
There is some weirdness going on here with the spacing by the looks of things, `description` `content` and `schema` should all be at the same indentation level I'd say.

Reply:
Not sure I'd agree because they're all parameters of the proceeding annotation, although I do prefer having the ending bracket on a new line.

Reply:
I do think there is something to be tidied up here, they are all parameters of the preceding annotation as you said but the indentation looks out to me. Not a major problem, just something to tidy up perhaps alongside everywhere else in the codebase that is in a similar state in a future PR.